student.rb

```diff
@@ -1,11 +1,12 @@
-class Student
+class Student 
+  ## I know that this should be a compositional relationship, but am not sure how to implement it. I need a module and don't have time for it.
```

Comment:
Didn't do release 0.  All 3 classes share common things:

```ruby
require_relative './hiFiveAble' 

class Person
  include HiFiveAble

  attr_reader :age, :phase
  attr_accessor :name

  def initialize(options = {})
    @age = options.fetch(:age, 0)
    @name = options.fetch(:name, "")
  end
end
```

This should use inheritance and inherit from a student class.  Not composition.  Only composition in that we need a module here.